loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -686,8 +687,30 @@ export class GLTFLoader implements IGLTFLoader {
         this.logClose();
 
         return Promise.all(promises).then(() => {
+            const min = TmpVectors.Vector3[0], max = TmpVectors.Vector3[1];
             this._forEachPrimitive(node, (babylonMesh) => {
-                babylonMesh.refreshBoundingInfo(true);
+                let recomputeBoundingInfo = true;
+                if (!this.parent.alwaysComputeBoundingBox && !babylonMesh.skeleton) {
+                    const tmp = GLTFLoader.GetTmpMetadata(babylonMesh);
+                    const mmin: [number, number, number] = tmp.positionMin, mmax: [number, number, number] = tmp.positionMax;
+                    if (mmin !== undefined && mmax !== undefined) {
+                        recomputeBoundingInfo = false;
+                        min.copyFromFloats(...mmin);
+                        max.copyFromFloats(...mmax);
+                        babylonMesh.getBoundingInfo().reConstruct(min, max);
+                        if (babylonMesh.subMeshes) {
+                            for (let index = 0; index < babylonMesh.subMeshes.length; index++) {
+                                babylonMesh.subMeshes[index].getBoundingInfo().reConstruct(min, max);
+                            }
+                        }
+                        babylonMesh._updateBoundingInfo();
+                    }
+                }
```

Comment:
Is it possible to do this as soon as the min/max is read to avoid using metadata?

Reply:
To avoid using metadata, I would need to add this code directly in the `loadAttribute` function.

But, can I be sure that the `babylonMesh.skeleton` property is up to date at that time, as I need it in the code above?

Reply:
Another problem is that `babylonGeometry.applyToMesh(babylonMesh)` is done later on (in relation to `loadAttribute`), and calling `applyToMesh` leads to creating the bounding info based on the current vertex data buffer, so overwrite the bouding box I create in `loadAttribute`.

Reply:
I have tried something, let me know what you think about the changes.

Note: it seems the `skeleton` property is indeed up to date, at least in my testing.